flow-typed/types-digitalAssets.js

```diff
@@ -0,0 +1,54 @@
+// @flow
```

Comment:
I suggest to place these files inside `flow-typed/redux` dir
so I would like to rename `flow-typed/types-digitalAssets.js` -> `flow-typed/redux/digitalAssets.js`

`types-digitalAssets` name looks really strange with dash and camelCase